biodatasets/__init__.py

```diff
@@ -4,14 +4,17 @@
 
 - CURRENT_DIRECTORY: directory with all the code
 
+- HOME_DIRECTORY: home directory
+
 - CACHE_DIRECTORY: directory used to store the cache datasets
 """
 from pathlib import Path
 
 CURRENT_DIRECTORY = Path(__file__).resolve().parent
 ROOT_DIRECTORY = CURRENT_DIRECTORY.parent
 
-CACHE_DIRECTORY = CURRENT_DIRECTORY / ".cache"
+HOME_DIRECTORY = Path.home()
+CACHE_DIRECTORY = HOME_DIRECTORY / ".cache/bio-datasets/"
```

Comment:
It won't work on windows if `/` is hardcoded between `.cache` and `bio-datasets`, we should let `pathlib` handle this
```suggestion
CACHE_DIRECTORY = HOME_DIRECTORY / ".cache" / "bio-datasets"
```

Reply:
Updated, thanks for the feedback @martinp7 !